### DISCUSSION TITLE
Issue with Quill.js webpack integration

### DISCUSSION THREAD

**Discussion by Marcelin97** [slab/quill#4148] [Q&A]
I'm encountering an issue while integrating Quill.js into my webpack setup. When trying to use Quill in my project, I'm getting the following error:

kotlin
Copy code
Uncaught Error: Module parse failed: Unexpected token (28:18)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| class Quill {
>   static DEFAULTS = {
|     bounds: null,
|     modules: {

It seems to be related to how webpack is handling Quill.js files, particularly with the static class field syntax used in Quill. 
I use Webpack with Encore in my symfony project.

I've tried various configurations and searched for solutions online, but I'm still encountering this issue. Could someone please provide guidance on how to resolve this error and properly integrate Quill.js into my webpack setup?

Thanks in advance for your help!

**Comment by ktoley** [comment]
Try this [Adding Quill to Your Build Pipeline](https://v1.quilljs.com/guides/adding-quill-to-your-build-pipeline)
The example project is very helpful.
